src/app/shared/loggerHelper.js

```diff
@@ -23,11 +23,15 @@ module.exports = {
       requestId: req.id,
     });
   },
-  logCoreBackCall: (req, { logCommunicationType, type, path, info }) => {
+  logCoreBackCall: (
+    req,
+    { logCommunicationType, type, path, info, context = "" },
+  ) => {
     const message = {
       logCommunicationType,
       path,
       info,
+      context,
     };
     if (type) {
       message.type = type;
```

Comment:
Is there a way we can include context in the request/response logging too? I think we might be able to if we store context in res.locals similar to sessionId - see: https://github.com/govuk-one-login/ipv-core-front/blob/eac294d0f93f75afcb2280f99d25fd92aeea54e3/src/lib/logger.js#L22